src/platform/amd/renoir/include/platform/drivers/interrupt.h

```diff
@@ -0,0 +1,173 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 AMD.All rights reserved.
+ *
+ * Author:      Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+ *              Anup Kulkarni <anup.kulkarni@amd.com>
+ *              Bala Kishore <balakishore.pati@amd.com>
+ */
+#ifdef __SOF_DRIVERS_INTERRUPT_H__
+
+#ifndef __PLATFORM_DRIVERS_INTERRUPT_H__
+#define __PLATFORM_DRIVERS_INTERRUPT_H__
+
+#include <sof/bit.h>
+#include <sof/lib/cpu.h>
+#include <sof/lib/memory.h>
+
+#define PLATFORM_IRQ_HW_NUM		8
+
+#define PLATFORM_IRQ_FIRST_CHILD	PLATFORM_IRQ_HW_NUM
+
+#define PLATFORM_IRQ_CHILDREN		0
+
+/* IRQ numbers - wrt Tensilica DSP */
+#define IRQ_NUM_SOFTWARE0	1	/* level 1 */
+
+#define IRQ_NUM_TIMER0		0	/* level 1 */
+
+#define IRQ_NUM_EXT_LEVEL3	3	/* level 1 */
+
+#define IRQ_NUM_TIMER1		6	/* level 2 */
+
+#define IRQ_NUM_EXT_LEVEL4	4	/* level 2 */
+
+#define IRQ_NUM_EXT_LEVEL5	5	/* level 3 */
+
+/* IRQ Masks */
+#define IRQ_MASK_SOFTWARE0	(1 << IRQ_NUM_SOFTWARE0)
+
+#define IRQ_MASK_TIMER0		(1 << IRQ_NUM_TIMER0)
+
+#define IRQ_MASK_TIMER1		(1 << IRQ_NUM_TIMER1)
+
+#define IRQ_MASK_EXT_LEVEL3	(1 << IRQ_NUM_EXT_LEVEL3)
+
+#define IRQ_MASK_EXT_LEVEL4	(1 << IRQ_NUM_EXT_LEVEL4)
+
+#define IRQ_MASK_EXT_LEVEL5	(1 << IRQ_NUM_EXT_LEVEL5)
+
+//==================AMD macros ====================================
+
+#define HOST_TO_DSP_INTR 	0x1
```

Comment:
If these are AMD macros then best to have an AMD_ prefix.

Reply:
Addressed the Review Comment. Done